### DISCUSSION TITLE
Changing the THR RED value on the PERF page gives "FORMAT ERROR" (exp fbw version)

### DISCUSSION THREAD

**Discussion by P1NBA11ER** [flybywiresim/aircraft#3357] [Help]
This was working last week but something changed with the MCDU and I can no longer enter the THR RED values without getting a "FORMAT ERROR" message. This is in the experimental FBW version. Do I need to enter the data differently? I'm trying to type in 860/3060 for example, to depart from KSNA using the noise abatement procedure.

**Comment by tracernz** [comment]
Try with a leading zero on the 860
![image](https://user-images.githubusercontent.com/9995998/107108413-89bd3f00-689c-11eb-97da-ece8827eff6c.png)

**Reply by P1NBA11ER** [reply]
It works... thanks! I watched a video of someone doing this and they didn't use a leading zero so I would have never thought to try this.
